Applications/Games/League of Legends/Online/script.js

```diff
@@ -94,12 +79,17 @@ var installerImplementation = {
             })
             .category("Games")
             .wineDistribution("staging")
-            .wineVersion(LATEST_STAGING_VERSION)
+            .wineVersion(LATEST_STAGING_VERSION) // TODO: Gallium9 patchset
```

Comment:
Is it really worth it with the current d3d9 -> OpenGL wine implementation ? This will aslo require the user to have drivers which support the gallium nine state tracker. I remember some times ago there was a discussion in wine-staging to include it with a check-box in winecfg. They did not do it at the end, because they did not want to maintain this. Maybe that could be a way so that everyone could play the game. However, that would mean @qparis have to be sure everything work in the builds.

Reply:
Do you have any guide on how to apply the patchset?

Reply:
@ImperatorS79 Based on my testing the game is more responsive and has more FPS. Aldo i would suggest agains including them atm assuming that Phoenicis's handling of WINE patchset is not efficient enough. 

Recommends going with currect commit and update if needed i do some additional testing since i think that the performance may be increased using DirectX10,10.1 or 11.